builtin/providers/google/resource_compute_firewall.go

```diff
@@ -23,6 +23,7 @@ func resourceComputeFirewall() *schema.Resource {
 			State: schema.ImportStatePassthrough,
 		},
 		SchemaVersion: 1,
+		MigrateState:  migrateFirewallStateV0toV1,
```

Comment:
I'm still learning go, so I apologize if I'm off here, but I think the reason this is breaking is the wrong function is being called. My guess is this should be [resourceComputeFirewallMigrateState](https://github.com/hashicorp/terraform/blob/master/builtin/providers/google/resource_compute_firewall_migrate.go#L13).